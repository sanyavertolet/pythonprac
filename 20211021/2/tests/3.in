:def A 0
def 2
:def A 1
def 2
:def z exp(z)*log(z)
def 2
:def quit quit*len(quit)+quit*3
def 'aboba'
:printhw 'Hello,World!!!'
printhw
:ab a b a*b
ab 2 4
:def a b c d a*b*c*d
def 1 2 3 4
quit '{}F{}'
