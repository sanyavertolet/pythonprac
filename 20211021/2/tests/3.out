0
1
5.121703401973049
abobaabobaabobaabobaabobaabobaabobaaboba
Hello,World!!!
8
24
'4F15'
